scriptmodules/ports/gmloader.sh

```diff
@@ -36,23 +42,54 @@ function build_gmloader() {
 }
 
 function install_gmloader() {
-    md_ret_files=('build/gmloader')
+    md_ret_files=(build/gmloader
+        LICENSE.md
+        README.md
+    )
+}
+
+# param1: filename of APK (either relative or absolute path)
+function addlib_gmloader() {
+    local apk="$(realpath "$1")"
+    printMsgs "console" "Adding libc++_shared.so to $(basename "$apk") ..."
+    apk_decoded_dir="${apk%.*}"
```

Comment:
For a temp dir that's about to get deleted later, I would just use 

    local tmpdir="$(mktemp -d)"